### PR TITLE
initial design

### PR DESCRIPTION
**Pull request by zspecza**
responsive template design for MAMA

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM iodigital/ubuntu-node-goddard:v1
+
+MAINTAINER Ant Cosentino <ant@io.co.za>
```
Comment (skibz) [inline Dockerfile:3]: not anymore, i'm not x)

File: views/partials/_header.jade [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            span.home-text Home
+            img(src='/img/home-icon.png').home-icon
+          for category, i in navbar_features
+            a(href="/#{category_path}/#{slugify(category.title)}.html"
+              class="#{i + 1 === navbar_features.length ? 'last' : ''}")= category.title
+  .langs
+    .wrapper
+      - reg = new RegExp('\/' + 'lang' + '\/' + current_locale + '\/')
+      for locale in locales
+        - translated_path = _path.replace(reg, '/lang/' + locale.isocode + '/').replace('index.html', '')
```
Comment (skibz) [inline views/partials/_header.jade:19]: `_path` is undefined, here.

Comment (zspecza) [inline views/partials/_header.jade:19]: Have you remembered to reinstall NPM deps? I fixed that issue here (we use a fork of roots-records with my fix, which has been merged, but not published to NPM yet): https://github.com/declandewet/roots-records/commit/b7c909c2838f1f485783287fa31382a847e460c6

**Comment by skibz** [comment]
verified working on my local development machine :+1:

**Pull request merged**